pkg/tar/tar_test.go

```diff
@@ -236,6 +278,18 @@ func verifyDirectory(t *testing.T, dir string, files []fileDesc) {
 					t.Errorf("File content is not equal for %s. Expected: %s, Actual: %s\n",
 						relpath, fd.content, content)
 				}
+				if isSymLink(fd.mode) {
+					target, err := os.Readlink(path)
+					if err != nil {
+						return err
+					}
+					if target != fd.target {
+						t.Errorf(
+							"Symbolic link %q points to wrong path. Expected: %s, Actual: %s\n",
+							fd.name, fd.target, target,
+						)
```

Comment:
just a small nit, I think it's more idiomatic in Go to have this as a single (long) line than write it in 4 lines. Don't know if we have project-specific preferences in this regard. @mfojtik any stylistic opinion?